data/clients.json

```diff
@@ -220,6 +220,26 @@
 			"url": "https://hub.docker.com/r/neilpang/letsproxy",
 			"category": "Docker"
 		},
+	        {
+			"name": "docker-openresty",
+			"url": "https://github.com/xiaojun207/docker-openresty⁠",
+			"category": "Docker",
```

Comment:
I believe these belong in the "nginx" category more than they belong in the docker category.

Reply:
Yes, I agree with your assessment. I have changed it to "nginx". Thank you.